src/core/public/chrome/utils.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AppCategory } from 'opensearch-dashboards/public';
+import { groupBy, sortBy } from 'lodash';
```

Comment:
Shall we avoid using `lodash`? As I have seen multiple issues about getting rid of `lodash`.

Reply:
Sure, updated to use native approach to group and sort.